### PR TITLE
Changes performed

### PR DESCRIPTION
**Pull request by hassanmehmud**
Blanks removed
Comments added
Couldn't update previous branch was giving errors so forked again

#### What does this PR do?
Dynamic messages added to console
API and console file updated for cases

#### Where should the reviewer start?
Console and API file

#### How should this be manually tested?
Normally use the console

#### Any background context you want to provide?
Relates Issue 34

#### What are the relevant tickets?
https://github.com/a-rmz/JEasyCrypto/issues/34

### CODE REVIEW

**Comment by hassanmehmud** [comment]
I couldn't update the previous branch because errors related to binaries, git was not allowing me to commit, so forked again and the commit was successful.

File: JEasyCrypto/src/easycrypto/EasyCryptoAPI.java [hunk 1]
```diff
@@ -24,9 +24,13 @@
 	private static final String currentVersion = "1.0.0";
 
 	public enum ResultCode {
-		ESuccess,      /*!< Text was successfully en/decrypted. */
-		EError,        /*!< There was an error with en/decryption. */
-		ENotSupported  /*!< The method of en/decription is not supported by the library */
+		ESuccess,      /*!< Text was successfully encrypted. */
```
Comment (a-rmz) [inline JEasyCrypto/src/easycrypto/EasyCryptoAPI.java]: Can you please align all the descriptions? I.e. same amount of tabs/spaces.

File: JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java [hunk 2]
```diff
@@ -22,39 +22,48 @@ public static void main(String[] args) {
 
 			while (true) {
 				String d = console.readLine("Do you wish to encrypt or decrypt (e or d)? > ");
-				String e = console.readLine("Please enter text to be encrypted > ");
-				String m = console.readLine("Please enter encryption method > ");
-
 				EasyCryptoAPI.Result result;
 
 				if (d.equalsIgnoreCase("e")) {
+				String e = console.readLine("Please enter text to be encrypted > ");
```
Comment (a-rmz) [inline JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java]: One indentation level missing in line 28 and 29.

File: JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java [hunk 3]
```diff
@@ -22,39 +22,48 @@ public static void main(String[] args) {
...
+				String m = console.readLine("Please enter encryption method > ");
 					result = EasyCryptoAPI.encrypt(e, m);
 				} else if (d.equalsIgnoreCase("d")) {
-					result = EasyCryptoAPI.decrypt(e, m);
+				String n = console.readLine("Please enter text to be decrypted > ");
```
Comment (a-rmz) [inline JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java]: One indentation level missing in line 32 and 33.

File: JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java [hunk 4]
```diff
@@ -22,39 +22,48 @@ public static void main(String[] args) {
...
+				String o = console.readLine("Please enter decryption method > ");
+					result = EasyCryptoAPI.decrypt(n, o);
 				} else {
 					return;
 				}
-		        
-				console.printf("\nResult is: %d - %s\n", result.resultCode().ordinal(), result.resultCode().toString());
 				switch (result.resultCode()) {
 				case ESuccess: {
```
Comment (a-rmz) [inline JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java]: Can you add one more level of indentation to all the cases please? For more clarity.
```java
switch (result.resultCode()) {
    case ESuccess: {
        // ...
    }
    case ...: {
        // ...
    }
}```

File: JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java [hunk 5]
```diff
@@ -22,39 +22,48 @@ public static void main(String[] args) {
...
+					console.printf("\nResult is: %d - %s\n", result.resultCode().ordinal(), result.resultCode().toString());
```
Comment (a-rmz) [inline JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java]: I think we should have a clearer message, `Result is: 0 - ESuccess` doesn't really say that much, but something like `Encryption successful` or something like that might be more understandable. Same thing with the decryption.

**Comment by hassanmehmud** [comment]
Conflicts are arising, not able to push resent changes.

**Pull request closed without merging**